getnacelle/nacelle-react
# Feature/cart local storage nc 1618

### WHY are these changes introduced?

For [[NC-1618](https://nacelle.atlassian.net/browse/NC-1618)]

### WHAT is this pull request doing?

This allows the cart's state is held in local storage. The developer can control whether or not they want to use this with the "useLocalStorage" prop on the CartProvider

### How to Test

Run `npm run dev` on the nextjs example.

Add/and or remove from your cart. Refresh the page and confirm the cart's state has persisted.
